company_website/admin.py

```diff
@@ -15,7 +15,7 @@ class EmployeesAdmin(AdminImageMixin, admin.ModelAdmin):
 
 class EstimateAdmin(admin.ModelAdmin):
```

Comment:
Commit which deletes these fields is lack of `[M]` on the beginning. We are always adding this on the beginning of the commit message to show that this commit has model changes

Reply:
Done